b2sdk/sync/sync.py

```diff
@@ -96,6 +96,32 @@ class KeepOrDeleteMode(Enum):
 
 
 class Synchronizer(object):
+    """
```

Comment:
the description doesn't mention the fact that we only take the newest version when handling bucket-to-bucket transfer, and we don't take old versions. It should be mentioned.

Reply:
It is in line 109:
>    The synchronizer can copy files:
>
>    ...
>    - Between different folders in the same B2 bucket.
>      **It will sync only the latest versions of files.**